components/Alert/alert.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/css/header.css">
+    <link rel="stylesheet" href="/css/main.css">
+    <link rel="stylesheet" href="alert.css">
+    <title>Alert</title>
+</head>
+
+<body>
+    <header class="header fixed-header">
+        <div class="header-left-icons">
+            <p class="header-heading">AMAZE-UI</p>
+        </div>
+        <div class="header-right-icons">
+            <a href="#" class="link">Documentation</a>
+        </div>
+    </header>
+    <div class="comp-main-container">
+        <nav class="navbar">
+            <p><a href="/index.html">Home</a></p>
+            <h3>Components</h3>
+            <ul class="nav-list">
+                <li class="nav-links"><a href="/components/Avatar/avatar.html">Avatar</a></li>
+                <li class="nav-links"><a href="#">Alert</a></li>
+                <li class="nav-links"><a href="/components/Badge/badge.html">Badge</a></li>
+                <li class="nav-links"><a href="#">Button</a></li>
+                <li class="nav-links"><a href="#">Card</a></li>
+                <li class="nav-links"><a href="#">Image</a></li>
+                <li class="nav-links"><a href="#">Input</a></li>
+                <li class="nav-links"><a href="#">List</a></li>
+                <li class="nav-links"><a href="#">Navigation</a></li>
+                <li class="nav-links"><a href="#">Toast</a></li>
```

Comment:
you can add navigation links of each component here ( or for at least the components you made ).

Reply:
done the changes